vespa-athenz/src/main/java/com/yahoo/vespa/athenz/client/zms/DefaultZmsClient.java

```diff
@@ -297,16 +297,15 @@ public boolean deletePolicyRule(AthenzDomain athenzDomain, String athenzPolicy,
 
         return roleEntity.roleMembers().stream()
                 .filter(RoleEntity.Member::pendingApproval)
-                .filter(re -> AthenzIdentities.USER_PRINCIPAL_DOMAIN.equals(AthenzIdentities.from(re.memberName()).getDomain()))
```

Comment:
Not sure why this was here to begin with, any possible side effects of removing this for the existing users of this method?

Reply:
I think it should be fine - all users of the function are only interested in finding whether a specific identity is pending.

@tokle was there any specific reason for the filter?

Reply:
Not sure, @andreer do you know?